### PR TITLE
28 updating the text viewer

### PR DESCRIPTION
**Pull request by cienco**

### CODE REVIEW

**Review by MPapini91 — CHANGES_REQUESTED** [review]
Opening Decameron with Edge and Firefox, I realized that the annotation labels are far away from the text, a distance that shrinks and disappears after a bit of scrolling. Can you run a check on this?
image

image

**Comment by cienco** [comment]
> Opening Decameron with Edge and Firefox, I realized that the annotation labels are far away from the text, a distance that shrinks and disappears after a bit of scrolling. Can you run a check on this? image
> 
> image

This problem is related to dirty data coming from backend. I wasn't able to reproduce this issue generating similar data using text editor. It's working correctly.

File: src/app/models/texto/paginated-response.ts
[the remark this answers is not in the document]
Comment (cienco) [reply src/app/models/texto/paginated-response.ts:1]: Ok I will fix this directly in the #27 as I moved some part of the code

**Comment by MPapini91** [comment]
> @cienco I noticed a problem with infinite scroll, you have a jump forward reading on the first scroll which is not correct. I have uploaded to Teams an example video on "Front-end"

@cienco  will you address this issue on this PR or on 27?

**Comment by cienco** [comment]
> > @cienco I noticed a problem with infinite scroll, you have a jump forward reading on the first scroll which is not correct. I have uploaded to Teams an example video on "Front-end"
> 
> @cienco will you address this issue on this PR or on 27?

yes I can see the problem I'll fix it in the #155

**Comment by cienco** [comment]
> > > @cienco I noticed a problem with infinite scroll, you have a jump forward reading on the first scroll which is not correct. I have uploaded to Teams an example video on "Front-end"
> > 
> > 
> > @cienco will you address this issue on this PR or on 27?
> 
> yes I can see the problem I'll fix it in the #155

Now should work

**Pull request merged**